locale/en_GB/LC_MESSAGES/django.po

```diff
@@ -290,3 +294,15 @@ msgstr "You must specify a year"
 #: judgments/views/advanced_search.py
 msgid "search.errors.to_before_from_detail"
 msgstr "Please enter a date after the 'from' date"
+
+#: transactional_licence_form/templates/layouts/form_step.html
```

Comment:
Do we need these translations, or should we just hard code the strings?

Reply:
previous decision: https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/adr/0015-translations.md